client/src/bin/greengrass-cli.ps1

```diff
@@ -0,0 +1,7 @@
+$bin = Split-Path $MyInvocation.MyCommand.Path -Parent
+$Env:CLI_HOME = Split-Path $bin -Parent
+$Env:GGC_ROOT_PATH = $Env:CLI_HOME + "\..\..\..\..\..\.."
+$CLI_JAR = $Env:CLI_HOME + "\lib\*"
+$CLI_LAUNCHER = "com.aws.greengrass.cli.CLI"
+
+java -classpath $CLI_JAR $CLI_LAUNCHER $args
```

Comment:
Powershell has execution policies which limit what customer can run. Please see [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.1) for detail and the defaults.

Reply:
I have removed the powershell implementation.